Cargo.toml

```diff
@@ -41,8 +41,8 @@ rustdoc-args = ["--document-private-items", "--generate-link-to-definition"]
 
 [dependencies]
 abscissa_core = { version = "0.7.0", default-features = false, features = ["application"] }
-rustic_backend = { git = "https://github.com/rustic-rs/rustic_core.git", features = ["cli"] }
-rustic_core = { git = "https://github.com/rustic-rs/rustic_core.git", features = ["cli"] }
```

Comment:
This is a good change I think.
Earlier some changes in `core` or `backend` could break  `rustic-rs` without even changing the code here.

Reply:
Yes. We do it that during development versions, rustic uses git dependencies on rustic_core and rustic_backend. This is because we dont't want to release each single improvement in those crates in order to have them available for rustic development versions.

However, when releasing rustic, we need version dependencies because git dependencies are not allowed by crates.io.

Reply:
Oh yeah so it's planned to revert to git dependencies after the release I see.